OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/WebViewActivity.java

```diff
@@ -11,7 +11,8 @@
 import android.webkit.WebResourceRequest;
 import android.webkit.WebResourceResponse;
 import android.webkit.WebView;
-import android.widget.ProgressBar;
+
```

Comment:
I think we don't need these files changes.

Reply:
update the references for code improvements.